mentoring/tests/integration/test_mcq.py

```diff
@@ -231,3 +260,11 @@ def test_questionnaire_html_choices(self, page):
         self.wait_until_disabled(submit)
 
         self.assertIn('Congratulations!', messages.text)
+
+
+@mock.patch.object(MentoringBlock, 'get_theme', mock.Mock(return_value={'package': None, 'locations': []}))
```

Comment:
@singingwolfboy The `MCQBlockBlankThemeTest` iherits the tests from `MCQBlockTest` - it runs the same tests, but without the `lms` theme, to make sure the tips are shown/hidden correctly when not using the `lms` theme.

Your comment made me realize that this was just a weird way to test the `apros` theme (which is currently essentially blank) - it would be better to tests against both supported themes (`lms` and `apros`) rather than against a "blank" theme, so I renamed the tests to `MCQBlockAprosThemeTest` and pathched it to run with the `apros` theme instead.